statsmodels/tsa/statespace/innovation.py

```diff
@@ -0,0 +1,72 @@
+from statsmodels.tsa.statespace.mlemodel import MLEModel, MLEResults, MLEResultsWrapper
+
+
+class InnnovationModel(MLEModel):
+    def __init__(self, endog, k_states, **kwargs):
+        kwargs.update(k_posdef=1)
+        k_states += 1
+        super().__init__(endog, k_states, **kwargs)
+        self.ssm['design', 0, -1] = 1
+        self.ssm['selection', -1, 0] = 1
+
+    def __setitem__(self, key, value):
+
+        _type = type(key)
+        if _type is str:
+            if key == 'cov':
+                self.ssm['state_cov', 0, 0] = value
+            elif key == 'transition':
+                self.ssm['transition', :-1, :-1] = value
+            elif key == 'selection':
+                self.ssm['transition', :-1, -1:] = value
```

Comment:
I don't see why trying to set the selection should set things in the transition matrix. Can you remind me which blog post? It could have been a typo on my part.

Given this, it looks like you never set the actual selection matrix, which I think would render the model useless? Or am I missing something?